dist/backdraft.js

```diff
@@ -1303,6 +1303,19 @@ _.extend(Plugin.factory, {
       }
     },
 
+    // takes a hash of { columnTitle: columnState, ... }
+    setColumnVisibilities: function(columns) {
+      _.each(columns, function(state, title) {
+        if (!state && this._columnManager.columnConfigForTitle(title).required) {
+          throw new Error("can not disable visibility when column is required");
+        }
+      }, this);
+      this._columnManager.visibility.set(columns);
```

Comment:
Bulk is exactly the whole reason I am doing this. Basically, when doing an _.each over the column visibility, every changed column triggers a "change" event, which triggers an ajax request for every column. When bulk changing, it triggers only one change event, and thus only one ajax request to the server.